ldap_probe/models.py

```diff
@@ -320,15 +341,59 @@ class LdapProbeLog(models.Model):
     is_expired = models.BooleanField(
         _('Probe data has expired'), db_index=True, blank=False, null=False,
         default=False)
+    failed = models.BooleanField(
+        _('Probe failed'), db_index=True, blank=False, null=False,
+        default=False)
 
     def __str__(self):
-        node = None
+        return f'LDAP probe {self.uuid} to {self.node}'
+
+    @property
+    def node(self):
+        """
+        return the node that was the target of this `LDAP` probe
+        """
         if self.ad_orion_node:
-            node = self.ad_orion_node.get_node()
-        else:
-            node = self.ad_node.get_node()
+            return self.ad_orion_node.get_node()
+
+        return self.ad_node.get_node()
+
+    @property
+    @mark_safe
+    def absolute_url(self):
+        """
+        absolute `URL` for the `Django admin` form for this instance
+
+        Note that there is no :class:`Django admin class
+        <django.contrib.admin.ModelAdmin>` matching this model. We will
+        have to use `admin views` pointing to the modles proxied from this
```

Comment:
I think you meant "modules"
```suggestion
        have to use `admin views` pointing to the modules proxied from this
```

Reply:
See above